types/react-native/index.d.ts

```diff
@@ -469,6 +471,84 @@ export interface Insets {
     right?: number;
 }
 
+export interface PressableStateCallbackType {
```

Comment:
Technically [the upstream type](https://github.com/facebook/react-native/blob/4f897336b6ce6c640a65f86b65c6d386ea55b944/Libraries/Components/Pressable/Pressable.js#L34) is read-only. I don’t feel very strong about this, but parity seems easy enough:

```suggestion
export type PressableStateCallbackType = Readonly<{
```